lib/dolly/timestamps.rb

```diff
@@ -0,0 +1,22 @@
+require 'active_support/concern'
```

Comment:
or... you can wrap the `tiemstamps!` method on a `classMethods` block and keep the Concern module